shell/browser/ui/inspectable_web_contents.cc

```diff
@@ -729,6 +729,11 @@ void InspectableWebContents::SetIsDocked(DispatchCallback callback,
     std::move(callback).Run(nullptr);
 }
 
+void InspectableWebContents::OnOpenItemComplete(const base::FilePath& path,
```

Comment:
This does not need to be a member method, it can be made a helper function in anonymous namespace.